cmd/version/version.go

```diff
@@ -41,15 +41,13 @@ type Version struct {
 	GoArch             string `json:"goArch"`
 }
 
-func GetVersion() Version {
```

Comment:
I wonder if we are doing this for release purpose. @droot can you confirm if this change is OK?

Reply:
This is used to print the version when user runs `kubebuilder version` command. This change doesn't buy much. Making it pkg private by renaming `getVersion` makes sense.